apiRoutes/routes.js

```diff
@@ -0,0 +1,51 @@
+const router = require('express').Router();
+const axios = require('axios');
+
+const robokache = require('../src/API/robokache');
+const handleAxiosError = require('../src/API/utils');
+
+const messenger = 'http://robokop.renci.org:4868';
+
+router.route('/answer')
+  .post(async (req, res) => {
+    const { questionId } = req.query;
+    try {
+      // TODO: do we first check to see if the question is owned by user?
```

Comment:
We shouldn't need to check this. Answer ownership is managed by robokache.

Reply:
I think this needs to be handled somewhere outside of Robokache. A signed in user could go to a non-owned public question and try and ask an ara. Then we have a user owning an answer to someone else's question, which I think is a bad idea.

Reply:
This ownership logic will be handled by the UI, so the server doesn't have to worry about it.